.github/workflows/Publish to Docker Hub.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         shell: pwsh
 
   docker-publish:
-    if: ${{ always() && env.DOCKER_HUB_TOKEN != '' }}
```

Comment:
Don't we still need to skip publishing when DOCKER_HUB_TOKEN is not available instead of failing this auto triggered build in forks.

Reply:
DOCKER_HUB_TOKEN is being checked in FindStandardAndExtensionVersions job

Reply:
As Discussed , I had separate step for env.DOCKER_HUB_TOKEN  for easy understanding

Reply:
env.DOCKER_HUB_TOKEN != '' if condition to FindStandardAndExtensionVersions job